scripts/docker/software_router/README.md

```diff
@@ -0,0 +1,8 @@
+This docker contains :
+
+- python environment installation
+- groovy installation
+- external software and scripts working with Cytomine
```

Comment:
This is no more true.
What is the purpose of this file ? I think it could be deleted as most of information are provided in repo README and 100% sure this file we never be updated ;)